apis/hive/v1/clusterdeployment_types.go

```diff
@@ -369,7 +369,9 @@ const (
 	// DeprovisionLaunchErrorCondition is set when a cluster deprovision fails to launch.
 	DeprovisionLaunchErrorCondition ClusterDeploymentConditionType = "DeprovisionLaunchError"
 
-	// ProvisionStoppedCondition is set when cluster provisioning is stopped
+	// ProvisionStoppedCondition is set when cluster provisioning is stopped.
+	// This indicates that at least one provision attempt was made, but there will be no further
+	// retries (without maxRetries changes or other hive configuration stopping futher retries).
```

Comment:
There's no such thing as `maxRetries` afaict. Should this refer to `InstallAttemptsLimit`?

Also typo "futher".
```suggestion
	// retries (without changing InstallAttemptsLimit or other hive configuration preventing further retries).
```